### PR TITLE
Remove what it is header from all Es docs

### PR DESCRIPTION
**Pull request by gamba47**
Describe your changes
fix https://github.com/cncf/glossary/issues/2575

Removing headers from Spanish documents

Replaces #2705

### CODE REVIEW

File: content/es/style-guide/_index.md [hunk 1]
```diff
@@ -159,8 +155,6 @@ Cuando corresponda, use **ejemplos del mundo real** que ayuden a los lectores (e
 
 Cuando se use en su definición, siempre **enlace a los términos existentes del glosario** (solo la primera mención debe tener un hipervínculo).
 
-**Ejemplo**: eche un vistazo a la sección "Qué es" de la [definición de service mesh](/es/service-mesh/).
-Se vincula con las definiciones de microservicios, servicio, fiabilidad y observabilidad.
```
Comment (jossemargt-cto-ai) [inline content/es/style-guide/_index.md:163]: I understand why you preferred to remove these lines, but, after checking on the EN source ([ref](https://github.com/cncf/glossary/blob/main/content/en/style-guide/_index.md?plain=1#L161-L162)) I see that what they did is to rephrase the sentence using "summary" to refer to the section instead. So, what do you think about applying this change instead?

```markdonw
**Ejemplo**: eche un vistazo a la introducción de la [definición de service mesh](/es/service-mesh/).
Se vincula con las definiciones de microservicios, servicio, fiabilidad y observabilidad.
```

Worth noting that "introducción" is not a direct translation for "summary", but I think it applies better in that context. Still, I think it is open to discussion.

Comment (gamba47) [reply content/es/style-guide/_index.md:163]: I agree with you.

**Review by electrocucaracha — APPROVED** [review]
/lgtm

**Review by jossemarGT — APPROVED** [review]
/lgtm

**Review by ramrodo — APPROVED** [review]
Thanks!

**Pull request merged**